Assets/ZSSRichTextEditor.js

```diff
@@ -1875,6 +2163,24 @@ ZSSEditor.closerParentNode = function() {
     return parentNode;
 };
 
+ZSSEditor.closerParentElementRelativeToNode = function(node) {
```

Comment:
No... there's a better alternative which is using the property `parentElement`.  I just removed this method.